src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -197,6 +212,18 @@ var DOMRenderer = ReactFiberReconciler({
     return true;
   },
 
+  commitMount(
+    domElement : Instance,
+    type : string,
+    newProps : Props,
+    rootContainerInstance : Container,
+    internalInstanceHandle : Object,
+  ) : void {
+    if (shouldAutoFocusHostComponent(type, newProps)) {
+      (domElement : any).focus();
```

Comment:
Aye, the PR summary noted that:

> #### IE8 compatibility note
> Note that this PR also drops use of the `focusNode` helper method since it was in place for IE8 and IE8 support was officially dropped in January 2016.

Seemed wrong to keep an abstraction around solely for compat with a browser version we no longer support.